README.md

```diff
@@ -217,6 +217,61 @@ We need to first add patient's details into wrapper's database. There are differ
          }
      ]'
      ```
+### Profile Share
+Adding of a patient in a particular facility
+- When the user scans and shares their profile, the wrapper generates a token and sends to the facility for notifying about the patient
+  - So the facility have to implement
+  * `POST` Request `/v1/profile/share`
+  ```
+    {
+      "token":"Wrapper generated token number for the user"
+      "hipId":"the HIP's counter code not the hip id"
+      "requestId": "499a5a4a-7dda-4f20-9b67-e24589627061",
+      "timestamp": "1970-01-01T00:00:00.000Z",
+      "intent": {
+        "type": "string"
+      },
+      "location": {
+        "latitude": "string",
+        "longitude": "string"
+      },
+      "profile": {
+        "hipCode": "12345 (CounterId)",
+        "patient": {
+          "healthId": "<username>@<suffix>",
+          "healthIdNumber": "1111-1111-1111-11",
+          "name": "Jane Doe",
+          "gender": "M",
+          "address": {
+            "line": "2nd cross street",
+            "district": "Chennai",
+            "state": "TamilNadu",
+            "pincode": "600301"
+          },
+          "yearOfBirth": 2000,
+          "dayOfBirth": 0,
+          "monthOfBirth": 0,
+          "identifiers": [
+            {
+              "type": "MOBILE",
+              "value": "9800083232"
+            }
+          ]
+        }
+      }
+    }
+  }
+  ```
+  - And the facility should respond with SUCCESS or FAILURE as status.
+  * Response :
+  ```
+  {
+    "status":"SUCCESS or FAILURE",
+    "healthId":"Patients ABHA number which you get in the request call",
+    "tokenNumber":"token generated from warpper which you get from /v1/profile/share"
```

Comment:
nit typo: should be wrapper instead of warpper

Reply:
I didn't notice it thanks.